### PR TITLE
[TBD] Test: not allow update system group

### PR DESCRIPTION
**Pull request by vmanilo**
## Changes
- added acc test to check error message on update system group

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -16,6 +16,8 @@ require (
 	gotest.tools/gotestsum v1.9.0
 )
 
+replace github.com/hashicorp/terraform-plugin-sdk/v2 v2.25.2 => github.com/vmanilo/terraform-plugin-sdk/v2 v2.25.2
```
Comment (ekampf) [inline go.mod:19]: I dont want to use a fork of the SDK so we need to find away around this

**Review by ekampf — CHANGES_REQUESTED** [review]
lets try and figure out a way that doesn't require terraform-sdk fork

File: go.mod (hunk 1 above)
Comment (vmanilo) [reply go.mod:19]: I've added PR to the terraform-plugin-sdk https://github.com/hashicorp/terraform-plugin-sdk/pull/1159

let's see if it be excepted and released

**Comment by vmanilo** [comment]
@ekampf FYI terraform guys extracted testing framework from sdk repo

https://discuss.hashicorp.com/t/announcing-the-terraform-plugin-testing-module/49914

they suggest to switch to new repo https://github.com/hashicorp/terraform-plugin-testing as it will be continue developing, and probably later there will be way to prevent destroy resource after tests.

so not sure, should we close this PR for now or just leave it as draft?

**Comment by vmanilo** [comment]
let's wait for PR with new option `RemoveState` https://github.com/hashicorp/terraform-plugin-testing/pull/118

**Comment by vmanilo** [comment]
PR to `terraform-plugin-testing` was closed https://github.com/hashicorp/terraform-plugin-testing/pull/118#issuecomment-1658843444

probably later this year terraform will have native support to unmanage resources, then we can get back to this test

**Pull request closed without merging**